src/plugins/github/parser.js

```diff
@@ -76,8 +76,11 @@ class GithubParser {
       | PullRequestReviewCommentNodePayload
       | PullRequestReviewNodePayload
     >,
-    authorJson: AuthorJSON
+    authorJson: NullableAuthorJSON
   ) {
+    if (authorJson == null) {
```

Comment:
Interesting that this is the only place where a check is required. I guess that that's a good thing!